docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+version: '2'
+services:
+  spark-master:
+    build: .
+    command: /usr/local/bootstrap.sh master
+    ports:
+      - "8080:8080"
+      - "7077:7077"
+      - "6066:6066"
+    environment:
+      - SPARK_MASTER_HOST=spark-master
+  spark-worker:
+    build: .
+    command: /usr/local/bootstrap.sh worker
+    links:
```

Comment:
would be very handy - only issue could be when scaling locally with `docker-compose scale` the port may already attached to a previous container. is it possible to give a range?